cmd/s3manager/main.go

```diff
@@ -36,6 +38,12 @@ func main() {
 	if !ok {
 		port = "8080"
 	}
+	basepath, ok := os.LookupEnv("BASE_PATH")
```

Comment:
This variable should be called `basePath` (instead of `basepath`) in order to be consistent with the naming of the env variable.